### PR TITLE
docs: Add some questions and answers to the FAQ

### PR DESCRIPTION
**Pull request by tzssangglass**
### What this PR does / why we need it:

Collate some common questions from the community and issue

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by tzssangglass** [comment]
@spacewander @tokers I used the new PR to improve the FAQ, and the relevant reivew comments in the previous PR have been modified.

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -552,3 +552,20 @@ apisix:
 **Note:**
 
 - Whenever trying to connect TLS services with cosocket, you should set `apisix.ssl.ssl_trusted_certificate`
```
Comment (spacewander) [inline docs/en/latest/FAQ.md:554]: Better to clarify that we only need to set trusted certificate when the certificate is untrusted.

File: docs/en/latest/FAQ.md [hunk 2]
```diff
@@ -552,3 +552,20 @@ apisix:
...
+
+## How to fix `module 'resty.worker.events' not found` error
+
+Installing APISIX under the `/root` directory causes this problem. Because the worker process is run by the user `nobody`, it does not have enough permissions to access the files in the `/root` directory. You need to change the APISIX installation directory, and it is recommended to install it in the `/usr/local` directory.
+
+## What is the difference between `plugin_metadata` and `plugin-configs`
+
+`plugin_metadata` is the metadata of the plugin, which is shared by all plugin instances. When writing a plugin, if there are some plugin properties that are shared by all plugin instances and the changes take effect for all plugin instances, then it is appropriate to put them in `plugin_metadata`.
+
+`plugin-configigs` is a collection of multiple plugin instances. If you want to reuse a common set of plugin configurations, you can extract them into a Plugin Config and bind them to the corresponding routes.
```
Comment (spacewander) [inline docs/en/latest/FAQ.md:566]:
```suggestion
`plugin-configs` is a collection of multiple plugin instances. If you want to reuse a common set of plugin configurations, you can extract them into a Plugin Config and bind them to the corresponding routes.
```

File: docs/en/latest/FAQ.md [hunk 3]
```diff
@@ -552,3 +552,20 @@ apisix:
...
 
 - Whenever trying to connect TLS services with cosocket, you should set `apisix.ssl.ssl_trusted_certificate`
+
+As an example, if using Nacos as a service discovery in APISIX, Nacos has TLS protocol enabled, i.e. Nacos configuration `host` starts with `https://`, so you need to configure `apisix.ssl.ssl_trusted_certificate` and use the same CA certificate as Nacos.
+
+## How to fix `module 'resty.worker.events' not found` error
+
+Installing APISIX under the `/root` directory causes this problem. Because the worker process is run by the user `nobody`, it does not have enough permissions to access the files in the `/root` directory. You need to change the APISIX installation directory, and it is recommended to install it in the `/usr/local` directory.
+
+## What is the difference between `plugin_metadata` and `plugin-configs`
```
Comment (spacewander) [inline docs/en/latest/FAQ.md:562]: We should have a consistent type for `-` & `_`.

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/FAQ.md [hunk 4]
```diff
@@ -549,4 +549,22 @@ apisix:
...
+
+举例：在 APISIX 中使用 Nacos 作为服务发现时，Nacos 开启了 TLS 协议， 即 Nacos 配置的 `host` 是 `https://` 开头，需要配置 `apisix.ssl.ssl_trusted_certificate`，并且使用和 Nacos 相同的 CA 证书。
+
+## 如何解决 `module 'resty.worker.events' not found` 错误
+
+在 `/root` 目录下安装 APISIX 会导致这个问题。因为 worker 进程的用户是 nobody，无权访问 `/root` 目录下的文件。需要移动 APISIX 安装目录，推荐安装在 `/usr/local` 目录下。
+
+## `plugin-metadata` 和 `plugin-configs` 有什么区别
+
+`plugin-metadata` 是插件的元数据，所有插件实例共享。在编写插件时，如果有部分插件属性，属于所有插件实例共享，修改对所有插件实例生效，那么放在 `plugin-metadata` 合适。
```
Comment (tokers) [inline docs/zh/latest/FAQ.md:563]: What is a "插件实例"

Comment (tzssangglass) [reply docs/zh/latest/FAQ.md:563]: update

File: docs/en/latest/FAQ.md [hunk 5]
```diff
@@ -569,3 +569,5 @@ The difference between `plugin-metadata` and `plugin-configs`:
 
 - Plugin instance scope: `plugin-metadata` works on all instances of this plugin. `plugin-configs` works on the plugin instances configured under it.
 - Binding entities: `plugin-metadata` take effect on the entities bound to all instances of this plugin. `plugin-configs` take effect on the routes bound to this `plugin-configs`.
+
+Note: plugin instances are plugins that are currently enabled
```
Comment (tokers) [inline docs/en/latest/FAQ.md:573]: I'm afraid the term `plugin instance` is not correct in this context, as the instance gives an illusion that there is more than one copy in Apache APISIX, but indeed we just have one object for each plugin.

Comment (tzssangglass) [reply docs/en/latest/FAQ.md:573]: > but indeed we just have one object for each plugin.

Here I do not understand. For example, the `limit-req` plugin on different routes, is it the same object? Also you are saying that each plugin has only one object, only the `conf` is different.

Comment (tzssangglass) [reply docs/en/latest/FAQ.md:573]: what about `plugin instance` -> `plugin conf`?

Comment (tzssangglass) [reply docs/en/latest/FAQ.md]: update

Comment (tokers) [reply docs/en/latest/FAQ.md]: OK.

**Review by tokers — APPROVED** [review]

**Pull request merged**